src/RequestLog/Middleware/LogRequest.php

```diff
@@ -129,7 +129,7 @@ private function logRequest(Request $request, Response $response): void
             ))->log(Log::getLogger());
 
         } catch (Throwable $throwable) {
-            Log::error($throwable);
+            Log::error($throwable->getTraceAsString());
```

Comment:
Do you want to discard the throwable message? Should we log the stracktrace as context, and message as message instead?